components/mgmtworker/scripts/snapshot_permissions_fixer

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+restservice_user=$(cat /usr/lib/systemd/system/cloudify-restservice.service | grep User | sed 's/User=//')
```

Comment:
Potentially- that'd require putting in extra relationships and pulling the info from there, and rather than doing that I figured this would probably be sensible since it can't get out of date even if someone slips up and makes an upgrade not work precisely as expected in future.